src/main/webapp/i18n/en/course.json

```diff
@@ -228,7 +228,11 @@
             "activeStudents": "Active students: {{ students }}",
             "notStartedYet": "The course has not officially started yet. Official course start:",
             "noOrganizations": "No Organizations",
-            "organizationAddingHint": "Administrators can add organizations to courses."
+            "organizationAddingHint": "Administrators can add organizations to courses.",
+            "imageCropper": {
+                "title": "Crop your image",
+                "uploadTitle": "Click to add"
```

Comment:
More of a nitpick: Maybe this text could be more explicit about what we're adding (-> an icon/logo). I'm not sure it's clear immediately what this button does when someone creates a course.

![image](https://github.com/ls1intum/Artemis/assets/38403547/4a74d014-de55-4f57-a7d4-20158ebbf701)

Reply:
I agree with this, however when we make the text longer it looks a bit clustered, especially German one (not sure about the translation).

<img width="455" alt="Screenshot 2024-04-17 at 23 26 07" src="https://github.com/ls1intum/Artemis/assets/23586149/9111d493-9768-4ef4-899a-a1b09c57c1ef">
<img width="455" alt="Screenshot 2024-04-17 at 23 26 29" src="https://github.com/ls1intum/Artemis/assets/23586149/20ea3f43-41fb-4ae6-ab07-e5db1f05693a">

 I'm not sure what can be a shorter alternative text for this. Do you have an idea?

Reply:
Hmm, you're right 🤔 

something like "Add an icon" / "Ein Bild hinzufügen"?

Reply:
I think this works. Changed